doc/news/DM-43823.bugfix.rst

```diff
@@ -0,0 +1 @@
+Add the missing descriptions of MTHexapod and MTRotator, and remove the MTHexapod-Telemetry and MTRotator-Telemetry from check_for_issues().
```

Comment:
In a way this indeed is a bugfix but the news item should be in `doc/news/interface_changes/DM-43823.mthexapod.rst` and `doc/news/interface_changes/DM-43823.mtrotator.rst` since it is related to those CSCs, as explained in the README.rst file. The bugfix and other categories refer to ts_xml bugfixes and other categories only.

Reply:
Updated in commit: [6302d15](https://github.com/lsst-ts/ts_xml/pull/825/commits/6302d154305884bdeea1a73628a5c00ff8a1af26). Thanks!